app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
@gaganawhad can you explain this a bit more? Do you have two separate paths that you're imagining a gem-using app might go with this?